### PR TITLE
[ADD] New module: web_flexnav

### PR DESCRIPTION
**Pull request by mscenna**
Replaces the current structure of smart buttons in the backend for a horizontal navigation that, if buttons exceed the 100% of the view, it dynamically generates a dropdown with the exceeding buttons.

### CODE REVIEW

File: web_flexnav/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# © 2016 Mercedes Scenna (bloopark systems GmbH)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    'name': 'Flexnav for Backend Smart buttons',
+    'summary': 'Smart buttons in Partner Views',
+    'author': "bloopark systems GmbH & Co. KG"
```
Comment (elicoidal) [inline web_flexnav/__openerp__.py]: add comma

File: web_flexnav/README.rst [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/web/issues>`_. In case of trouble, please
+check there if your issue has already been reported. If you spotted it first,
+help us smashing it by providing a detailed and welcomed `feedback
+<https://github.com/OCA/
```
Comment (elicoidal) [inline web_flexnav/README.rst]: remove this part following latest template

File: web_flexnav/README.rst [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+This module adds a dynamic navigation that contains the smart buttons in the backend views. The need came from having
+a better use of the space.
+
+The javascript implementation is strongly based on https://github.com/352Media/flexMenu
+
+Usage
+=====
+
+You should move the smart buttons to one line on top of the view, instead of floating on the right since when 
+the number of buttons increases, the content taht follows in the same view is pushed further down.
```
Comment (elicoidal) [inline web_flexnav/README.rst]: s/taht/that

File: web_flexnav/README.rst [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+You replace the existing smart buttons with the code snippet that have the JavaScript classes (done 
+in res_partner.xml). 
+
```
Comment (elicoidal) [inline web_flexnav/README.rst:22]: Screenshot would be a good nice to have

File: web_flexnav/__openerp__.py [hunk 5]
```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# © 2016 Mercedes Scenna (bloopark systems GmbH)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    'name': 'Flexnav for Backend Smart buttons',
+    'summary': 'Smart buttons in Partner Views',
```
Comment (elicoidal) [inline web_flexnav/__openerp__.py]: is it only for partner view?
My point is if so please specify in the README as well

File: web_flexnav/static/src/js/backend_smart_button.js [hunk 6]
```diff
@@ -0,0 +1,144 @@
+openerp.web_flexnav = function (instance) {
```
Comment (elicoidal) [inline web_flexnav/static/src/js/backend_smart_button.js:8]: Please add copyright notice

**Comment by elicoidal** [comment]
Thanks for the contribution!

**Comment by dreispt** [comment]
Travis needs love!

**Review by susport — COMMENTED** [review]
Please migrate the code to the new api and correct coding standard errors that forces travis.

**Pull request closed without merging**